python/ray/serialization.py

```diff
@@ -354,6 +357,14 @@ def _serialize_to_msgpack(self, value):
             metadata = str(
                 ErrorType.Value("TASK_EXECUTION_EXCEPTION")).encode("ascii")
             value = value.to_bytes()
+        elif isinstance(value, ray.actor.ActorHandle):
```

Comment:
Why not put the logic into `actor_handle_reducer`?

Reply:
`actor_handle_reducer` is for cloud pickle (e.g. a list of ActorHandle passed from Python to Python), while this logic is for cross language serialization.

Reply:
I see. But can you try to refactor them to avoid duplicated code?

Reply:
I think it's not necessary to merge them together:
- Only one line is duplicated: `serialized, actor_handle_id = value._serialization_helper()`
- Current implementation is not the final version, the logic will be changed after we have `custom cross-language type` feature.